src/components/manage/Blocks/Block/BlocksForm.jsx

```diff
@@ -161,6 +173,12 @@ const BlocksForm = (props) => {
     }
   }
 
+  useEvent('voltoClickBelowContent', () => {
```

Comment:
There can be multiple BlocksForm on a screen, at one moment. The BlocksForm is also used by the "blocks container" addons, the accordion, column, tabs, etc.

I propose we move this to the main Edit / Add form, if feasable.

Reply:
@tiberiuichim Thanks, that is a helpful hint. I think handling the event needs to be here in BlocksForm, so that we have access to `onAddBlock` and `onSelectBlock`. But it looks like I can use the `isMainForm` prop to make sure that only the top-level BlocksForm listens for the event. Right?

Reply:
Right. I forgot that that flag exists.

Reply:
@tiberiuichim https://github.com/plone/volto/commit/a39b81a50b0065ca1b8c8901d795429ca9cb7ec8

Reply:
WP always have a virtual "placeholder" after the last block. It's invisible until you hover over it. In mobile, we could show it always.